index.html

```diff
@@ -1,29 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
-</head>
-<body>
-    <h1 class="heading">All Awesome Books</h1>
-    <div class="showBooks" id="showBooks"></div>
-    <hr class="hr-line">
-    <div class="container">
+  </head>
+  <body>
+    <header>
+      <nav>
+        <p>Awesome Books</p>
+        <ul class="link">
+          <li><a href="#" class="list-nav">List</a></li>
+          <li>|</li>
+          <li><a href="#" class="books-nav">Add new</a></li>
+          <li>|</li>
+          <li><a href="#" class="contact-nav">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section>
+      <p id="time"></p>
+    </section>
+    <section>
+      <div class="book-list">
+        <h1 class="heading">All Awesome Books</h1>
+        <div class="showBooks" id="showBooks"></div>
+      </div>
+      <div class="container add-books">
         <h2 class="heading">Add A New Book</h2>
         <form action="#" id="my-form">
-            <label for="title">
-                <input type="text" id="title" placeholder="Title" required>
-            </label>
-            <br>
-            <label for="author">
-                <input type="text" id="author" placeholder="Author" required>
-            </label>
-            <button type="submit" id="addBtn">Add</button>
+          <label for="title">
+            <input type="text" id="title" placeholder="Title" required />
+          </label>
+          <br />
+          <label for="author">
+            <input type="text" id="author" placeholder="Author" required />
+          </label>
+          <button type="submit" id="addBtn">Add</button>
         </form>
-    </div>
-<script src="file.js"></script>
-</body>
-</html>
\ No newline at end of file
+      </div>
+    </section>
+    <section class="contact">
+      <div class="info">
+        <h2>Contact Information</h2>
+        <p>
+          Do you have any questions or do just want to say hello?<br />
+          You can reach out to us
+        </p>
+        <ul>
+          <li>completewebsite@gmail.com</li>
+          <li>Our phone number: 3632809218</li>
+          <li>Our address: 74 Kimathi street</li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ] There is nothing wrong with this section. It is just an emphasis that we ensure we stick the footer to the bottom section of the page as in the wireframes. Please work on this. Screenshots:

| Wireframe | Implementation |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/45002654/153282419-861f0ead-ff59-467d-8a2f-f7498def9e68.png) | ![image](https://user-images.githubusercontent.com/45002654/153282483-f3790ff7-a03c-4f2a-be61-dd5d66887985.png) |

Reply:
Thanks for your review @Lameck1 . We have worked on that and solved it.